README.rst

```diff
@@ -148,22 +148,22 @@ If you intend to use json-support you also need
 Target Prerequisites
 ~~~~~~~~~~~~~~~~~~~~
 
-Required kernel options:
-
--  ``CONFIG_MD=y``
--  ``CONFIG_BLK_DEV_DM=y``
--  ``CONFIG_BLK_DEV_LOOP=y``
--  ``CONFIG_DM_VERITY=y``
--  ``CONFIG_SQUASHFS=y``
--  ``CONFIG_CRYPTO_SHA256=y``
+Required kernel options (`y` or at least `m`)::
+
+-  ``CONFIG_MD``
+-  ``CONFIG_BLK_DEV_DM``
+-  ``CONFIG_BLK_DEV_LOOP``
+-  ``CONFIG_DM_VERITY``
+-  ``CONFIG_SQUASHFS``
+-  ``CONFIG_CRYPTO_SHA256``
 -  ``CONFIG_BLK_DEV_NBD`` (for streaming support)
 -  ``CONFIG_DM_CRYPT`` (for encryption support)
 
 For using tar archive in RAUC bundles with Busybox tar, you have to enable the
 following Busybox feature:
 
--  ``CONFIG_FEATURE_TAR_AUTODETECT=y``
```

Comment:
What does Kconfig `=m` map to in busybox? I'd just leave this as is as it's unrelated to your kernel configuration.

Reply:
All this change does is to allow not only `y`, but also `m`. So why does it matter to what it maps on busybox?

Reply:
The commit message claims it's about kernel options, which busybox configuration is not. You can write a proper commit message and say something along the lines of "While we are at it, change required busybox CONFIG_ variables for symmetry" and that would be ok.

Reply:
Busybox doesn't use `=m`, so changing it should stay as `=y`. Anything else would just be confusing. :)